src/pages/Home/Home.page.jsx

```diff
@@ -1,38 +1,17 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React /*, { useRef }*/ from 'react';
+// import { useHistory } from 'react-router-dom';
```

Comment:
**No big deal**:
if we're not using them, it is cleaner to remove code instead of commenting it out.  If you want to keep it as reference I recommend a new file `reference.txt` or something similar

Reply:
Ok then. I was thinking about leaving them there, beause I have no clue whether they will be useful in the future, or how to use them hehe. So having those lines there as a guide, might be useful.

Your idea of keeping those things in a separate file is better though. I think I will do that. Thanks.